content/en/docs/concepts/cluster-administration/networking.md

```diff
@@ -293,6 +293,78 @@ Weave Net runs as a [CNI plug-in](https://www.weave.works/docs/net/latest/cni-pl
 or stand-alone.  In either version, it doesn't require any configuration or extra code
 to run, and in both cases, the network provides one IP address per pod - as is standard for Kubernetes.
 
+## IPv4/IPv6 dual stack
+
+{{< feature-state state="alpha" >}}
+{{< warning >}}Kubernetes' implementation of dual-stack networking is in flux. You may well find that the design is changed in future releases.{{< /warning >}}
+
+IPv4/IPv6 dual stack enabled Kubernetes clusters allow the assignment of both IPv4 and IPv6 addresses simultaneously which enables the following capabilities:
+
+   * Awareness of multiple IPv4/IPv6 address assignments per Pod
+   * Native IPv4-to-IPv4 in parallel with IPv6-to-IPv6 communications to, from, and within a cluster
+
+Details on the implementation of this feature may be found in the [IPv4/IPv6 dual stack KEP](https://github.com/kubernetes/enhancements/blob/master/keps/sig-network/20180612-ipv4-ipv6-dual-stack.md)
+
+### Supported Features
+
+Enabling IPv4/IPv6 dual stack on a Kubernetes cluster enables the following features:
+
+   * Dual stack Pod networking (multi-IP Pod)
+   * Kubenet multi address family support (IPv4 and IPv6)
+   * Egress Pod Internet routing via IPv4/IPv6 addresses
+
+### Prerequisites
+
+The following prerequisites are needed in order to utilize IPv4/IPv6 dual stack Kubernetes clusters:
+
+   * Provider support for dual stack networking (Cloud provider or otherwise must be able to provide Kubernetes nodes with routable IPv4/IPv6 network interfaces)
+   * Kubenet network plugin
+
+### Enable IPv4/IPv6 dual stack
+
+To enable IPv4/IPv6 dual stack, the following flags must added to these cluster components:
```

Comment:
To help the reader, link to the page on feature gates.

```suggestion
To enable IPv4/IPv6 dual stack, enable the `IPv6DualStack` [feature gates](https://kubernetes.io/docs/reference/command-line-tools-reference/feature-gates/] for the relevant components of your cluster, and set dual-stack cluster network assignments:
```

NB: I intentionally didn't write CIDR in this suggested text; `--cluster-cidr` is a slight misnomer, as CIDR is an IPv4 concept.